components/Nav.vue

```diff
@@ -0,0 +1,336 @@
+<template>
+  <nav>
+  <div class="hamburger-menu">
+    <input id="menu__toggle" type="checkbox" />
+    <label class="menu__btn" for="menu__toggle">
+      <span></span>
+    </label>
+
+    <ul class="nav-links" > 
+        <li class="pages"><a href="#">Home</a></li>
+        <li class="pages"><a href="#">Shop</a></li>
+        <li class="pages"><a href="#">About Us</a></li>
+        <li class="get_it"><a href="#"><img src="get_it.png"><span>GetIt</span></a></li> 
+        <li class="pages"><a href="#" class="sign_in">Sign In</a></li>
+        <li class="cart"><a href="#">Cart <img src="cart.png"></a></li>                
+    </ul>
+    </div>
+    
+    <li class="search"><a href="#"><input type="text"  class="s-btn" placeholder="Search...">
+    <svg stroke="currentColor" fill="grey" stroke-width="0" viewBox="0 0 512 512" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"><path d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z">
+    </path></svg></a>
+    </li>
```

Comment:
@whizzy23 Can you please explain what this peice of code does ?

Reply:
It is used to bring search icon